### DISCUSSION TITLE
Multiple windows with different views of the same Neovim instance using –remote-ui?

### DISCUSSION THREAD

**Discussion by smackesey** [neovim/neovim#29994] [Q&A] [answered]
0.9.x brought the --remote-ui option, which allows for opening two OS terminal windows against the same Neovim instance. But is it possible to have those OS windows have different “views” (i.e. different collections of vim tabpages/windows)?

My primary motivation is to run a debugger and show the UI in a separate OS window/display.

**Comment by zeertzjq** [comment, marked as answer]
This is not achievable using `--remote-ui`. This is only achievable with an ext_multigrid GUI (e.g. goneovim) using external floating windows (using methods mentioned in `:h nvim_open_win()` or `:h CTRL-W_ge`).

**Reply by smackesey** [reply]
Thanks. For those looking for more info, I inquired what is possible in goneovim with its maintainer: https://github.com/akiyosi/goneovim/discussions/546#discussion-7021815

TLDR you can break out a single vim window (i.e. pane) into an "external floating window" (i.e. separate OS window) but the current architecture of Neovim prevents having two full-featured multi-paned windows/UIs opened against the same Neovim instance.
